st2reactor/st2reactor/rules/filter.py

```diff
@@ -149,7 +154,36 @@ def _render_criteria_pattern(self, criteria_pattern):
             # makes no sense
             return criteria_pattern
 
-        criteria_pattern = render_template_with_system_context(value=criteria_pattern)
+        LOG.debug(
+            'Rendering criteria pattern (%s) with context: %s',
+            criteria_pattern,
+            criteria_context
+        )
+
+        to_complex = False
+
```

Comment:
Can you provide description here why we are looking for jinja templates here and what is the expected outcome for the substitution? It's not straightforward to understand.